cmake/modules/AddSwift.cmake

```diff
@@ -486,11 +486,8 @@ function(_add_swift_lipo_target)
         DEPENDS ${source_targets})
   else()
     # We don't know how to create fat binaries for other platforms.
-    add_custom_command_target(unused_var
-        COMMAND "${CMAKE_COMMAND}" "-E" "copy" "${source_binaries}" "${LIPO_OUTPUT}"
-        CUSTOM_TARGET_NAME "${LIPO_TARGET}"
-        OUTPUT "${LIPO_OUTPUT}"
-        DEPENDS ${source_targets})
+    message(FATAL_ERROR
+        "It's not possible to build a universal binary for a non-Darwin platform")
```

Comment:
This is a MachO feature, not a Darwin feature I believe.

Reply:
Darwin being the only platform that has a lipo like tool though to actually make multi-arch binaries :) 

Technically there is the FatELF thing but it's niche and pretty much dead.

Reply:
There also have been other projects to run MachO binaries on Linux.  In such a case, a `lipo` tool could work on Linux just as well.  It really is something which is part of the MachO binary format, and it just happens that Darwin has the most stable support for MachO binaries.